frontend/packages/knative-plugin/src/components/overview/KnativeResourceOverviewPage.tsx

```diff
@@ -33,14 +36,17 @@ interface StateProps {
 
 export interface KnativeResourceOverviewPageProps extends StateProps {
   item?: OverviewItem;
+  element?: GraphElement;
```

Comment:
no need to pass item if you passing element.

Although, This component will be removed once all the knative resource are migrated

Reply:
yes, it will get removed